src/index.html

```diff
@@ -1,13 +1,65 @@
+<!-- #region head -->
+
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <div class="card page__card" data-qa="card">
```

Comment:
Remove `page` BEM block
![image](https://github.com/mate-academy/layout_product-cards/assets/70009205/bc72760d-bc80-446f-bacc-a44c1350dbf3)

```suggestion
    <div class="card" data-qa="card">
```